plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -33,8 +33,33 @@
 #define FU_PXI_DEVICE_CMD_FW_OTA_DISCONNECT	0x29
 
 #define FU_PXI_DEVICE_OBJECT_SIZE_MAX		4096	/* bytes */
-#define FU_PXI_DEVICE_OTA_PAYLOAD_SZ		20	/* bytes */
 #define FU_PXI_DEVICE_OTA_BUF_SZ		32	/* bytes */
+#define FU_PXI_DEVICE_NOTTFY_RET_LEN		4	/* bytes */
+#define FU_PXI_DEVICE_fW_IFNO_RET_LEN		8	/* bytes */
```

Comment:
`FW_INFO`, surely?

Reply:
Do you have any good idea for naming ? Thank you

Reply:
I will fix to INFO